.github/workflows/golden_tests.yml

```
@@ -0,0 +1,101 @@
+      - name: Install dependencies
+        run: flutter pub get
+
+      # First run to generate baseline golden files
+      - name: Generate golden files
+        run: flutter test --update-goldens --tags golden
+        continue-on-error: true # Don't fail if there are no golden tests yet
+      
+      # Run tests to check for differences
+      - name: Run golden tests
+        id: golden-test-run
+        run: |
```

actionlint (1.7.4)

workflow command "set-output" was deprecated. use `echo "{name}={value}" >> $GITHUB_OUTPUT` instead: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions

(deprecated-commands)